firefox_ui_tests/firefox_ui_tests/functional/security/test_safe_browsing_primary_download.py

```diff
@@ -0,0 +1,88 @@
+import os
+import sys
+
+from marionette_driver import By, Wait
+from marionette.marionette_test import skip_if_e10s
+
+from firefox_puppeteer.testcases import FirefoxTestCase
+
+from firefox_puppeteer.ui.browser.window import BrowserWindow
+
+
+class TestSafeBrowsingPrimaryDownload(FirefoxTestCase):
+
+    data = [
+        {
+            'platform': ['linux', 'win', 'darwin'],
+            'files': [
+                # Phishing
+                "goog-phish-shavar.cache",
+                "goog-phish-shavar.pset",
+                "goog-phish-shavar.sbstore",
+                "goog-malware-shavar.cache",
+                "goog-malware-shavar.pset",
+                "goog-malware-shavar.sbstore",
+                "goog-badbinurl-shavar.cache",
+                "goog-badbinurl-shavar.pset",
+                "goog-badbinurl-shavar.sbstore",
+
+                # Tracking Protections
+                "mozstd-track-digest256.cache",
+                "mozstd-track-digest256.pset",
+                "mozstd-track-digest256.sbstore",
+                "mozstd-trackwhite-digest256.cache",
+                "mozstd-trackwhite-digest256.pset",
+                "mozstd-trackwhite-digest256.sbstore"
+                ]
+            },
+        {
+            'platform': ['win'],
+            'files': [
+                "goog-downloadwhite-digest256.cache",
+                "goog-downloadwhite-digest256.pset",
+                "goog-downloadwhite-digest256.sbstore"
+            ]
+        }
+    ]
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        # Set Browser URL
+        self.test_url = 'https://mozqa.com'
+
+        # Set Browser Preferences
+        self.prefs.set_pref('browser.safebrowsing.provider.google.lastupdatetime', 1)
+        self.prefs.set_pref('browser.safebrowsing.provider.google.nextupdatetime', 1)
+        self.prefs.set_pref('browser.safebrowsing.enabled', 'true')
+        self.prefs.set_pref('browser.safebrowsing.malware.enabled', 'true')
```

Comment:
These two prefs default to `true`, but if you want to be explicit, I would also do:
```
browser.safebrowsing.downloads.enabled = true
browser.safebrowsing.downloads.remote.enabled = true
privacy.trackingprotection.pbmode.enabled = true
```